protobuf/flow/executiondata/executiondata.proto

```diff
@@ -63,6 +66,70 @@ service ExecutionDataAPI {
   rpc SubscribeEvents(SubscribeEventsRequest)
       returns (stream SubscribeEventsResponse);
```

Comment:
I think this is the syntax to mark the RPC deprecated
```suggestion
      returns (stream SubscribeEventsResponse) {
      option deprecated = true;
  };
```

Reply:
[bc18dca](https://github.com/onflow/flow/pull/1441/commits/bc18dca2e2f7ffd0a18b2ea3e3523483b59b1752)